pkg/microservice/aslan/core/workflow/service/webhook/utils.go

```diff
@@ -705,3 +722,16 @@ func getServiceSrcPath(service *commonmodels.Service) (string, error) {
 	_, _, _, _, p, _, err := GetOwnerRepoBranchPath(service.SrcPath)
 	return p, err
 }
+
+// check if sub path is a part of parent path
+func subElem(parent, sub string) bool {
+	up := ".." + string(os.PathSeparator)
+	rel, err := filepath.Rel(parent, sub)
+	if err != nil {
```

Comment:
Return a special error?

Reply:
this purpose of this function is quite simple and there is no need to return err
i will log the error here

Reply:
this purpose of this function is quite simple and there is no need to return err
i will log the error here